.c8rc.json

```diff
@@ -1,4 +1,5 @@
 {
-  "reporter": ["html", "lcov"],
-  "exclude": ["test/*", "*.mjs", "**/*.mjs"]
+  "reporter": ["lcov"],
```

Comment:
TIL that lcov creates both lcov and html reports. So got rid of the redundant report as it was actually creating 2 html reports